ipaclient/install/client.py

```diff
@@ -1002,6 +1002,11 @@ def configure_sssd_conf(
 
     if options.dns_updates:
         domain.set_option('dyndns_update', True)
+        if options.dns_over_tls:
+            server_ip = str(list(dnsutil.resolve_ip_addresses(
+                cli_server[0]))[0])
+            domain.set_option('dyndns_server', 'dns+tls://{}:853#{}'
+                              .format(server_ip, cli_server[0]))
```

Comment:
You also need to pass --dns-over-tls-cert and --dns-over-tls-key to sssd configuration.  See these options: https://github.com/SSSD/sssd/blob/d2232139a2b9453f28824714c8b04b9e9f554df9/src/man/sssd-ipa.5.xml#L367-L414